components/proxy/conf/Caddyfile

```diff
@@ -223,6 +223,37 @@ https://{$GITPOD_DOMAIN} {
 		}
 	}
 
+	@local_app {
+		path /static/bin/gitpod-local-companion-linux
```

Comment:
It'd be awesome if we could reduce this to a single line a la
`path /static/bin/gitpod-local-companion*`
to minimize the dependency.

Reply:
@geropl Already changed CC @akosyakov

Reply:
Ok. The change without the `*` here would not work (exact match by default), but I see [you already fixed it](https://github.com/gitpod-io/gitpod/pull/7869/commits/5ebec75cd3ec4d246e5cd65c8cf3cb0f9a3987e8#diff-e64d416767db1a1dc93494d871291f34493dd9de076ea60552167d506c083c9aR227). :heavy_check_mark: